cmd/gpu_plugin/README.md

```diff
@@ -154,7 +154,7 @@ resource allocation status on the nodes:
 
 ```bash
 $ kubectl get nodes -o=jsonpath="{range .items[*]}{.metadata.name}{'\n'}{' i915: '}{.status.allocatable.gpu\.intel\.com/i915}{'\n'}"
-master
+main
```

Comment:
master remains here

Reply:
Restored master to line 157. See commit to close 560.